src/AgGrid.elm

```diff
@@ -520,6 +522,8 @@ Can be used when implementing column configurations for the table.
 Default column settings:
 
     { aggFunc = NoAggregation
+    , allowedAggFuncs = []
```

Comment:
We need to update the comment now.

Otherwise LGTM

Reply:
:facepalm: Done